assets/js/vue-apps/components/facet-choice.vue

```diff
@@ -28,6 +28,12 @@ export default {
         },
         fieldId(index) {
             return `field-dynamic-${this.name}-${index}`;
+        },
+        toggle() {
+            this.isOpen = !this.isOpen;
+            let action = 'Toggle additional options ';
+            action += this.isOpen ? 'on' : 'off';
+            this.trackUi(action, this.name);
```

Comment:
You might be able to do this declaratively using `data-ga-*` events in the markup.

Reply:
I think I prefer leaving them as-is for now as the some of the attributes are a bit tricky to define in one-liners (eg. the above), and it'll mean hardcoding strings like `Past Grants Interface` everywhere – I like having the two tracking functions defined next to each other in one place.